packages/wrangler/src/api/form_data.ts

```diff
@@ -88,6 +94,15 @@ export function toFormData(worker: CfWorkerInit): FormData {
     metadataBindings.push({ name: key, type: "plain_text", text: value });
   });
 
+  bindings.services?.forEach(({ name, service, environment }) => {
+    metadataBindings.push({
+      name,
+      type: "service",
+      service,
```

Comment:
```suggestion
      script_name: service,
```
I think the API requires `script_name`, which we can change later.

Reply:
wait, which one is it, @danielrs said it's `service` here haha https://github.com/cloudflare/wrangler2/pull/166#discussion_r774736845

Reply:
I'm going to land this, so y'all can test it out.